proto/warden/sched/v1beta1/extension.proto

```
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package warden.sched.v1beta1;
+
+option go_package = "github.com/warden-protocol/wardenprotocol/warden/x/sched/types/v1beta1";
+
+message ExtensionOptionsCallbacks {}
```

Buf (1.54.0)

Message "ExtensionOptionsCallbacks" should have a non-empty comment for documentation.

(COMMENT_MESSAGE)